AcadeMeData/tests/test_course.py

```diff
@@ -1,22 +1,34 @@
 import pytest
-from AcadeMeData.models import Course
+from AcadeMeData.models import Course, Degree
 
 
 @pytest.fixture
-def generate_course(generate_degree, generate_professor, course_id=1, name="History of Countries",
+def generate_course(generate_degree, generate_professor, generate_university, course_id=1, name="History of Countries",
                     mandatory=True, description="Learn about historic events and their influences on countries"):
-    professor = generate_professor
-    degree = generate_degree
-    course = Course.create_course(course_id=course_id, name=name, degree=degree, mandatory=mandatory,
-                                  description=description, professor=professor)
+    course = Course.create_course(course_id=course_id, name=name, degree=generate_degree, mandatory=mandatory,
+                                  description=description, professor=generate_professor, university=generate_university)
     course.save()
     return course
 
 
+@pytest.fixture
+def generate_degree2(degree_id=5, name='Computer Science', universities="Reichman University",
```

Comment:
Please give it a more informative name. what differs between this and `generate_course`? (I know the answer, write it so other readers will know too)

Reply:
Done @Omeramsc